invenio_theme/templates/semantic-ui/invenio_theme/page_settings.html

```diff
@@ -11,7 +12,7 @@
 
 {%- block page_body scoped %}
 <div class="ui grid container stackable">
-  <div class="four wide column">
+  <nav class="four wide column" aria-label="Account settings">
```

Comment:
minor: should aria-labels be translated?  e.g. `aria-label="{{ _('Account settings') }}"` ?

Reply:
Perfect, thanks! Wasn't sure how to handle that.